api/types/stats.go

```diff
@@ -170,6 +170,9 @@ type Stats struct {
 type StatsJSON struct {
 	Stats
 
+	Name string `json:"name,omitempty"`
+	ID   string `json:"id,omitempty"`
```

Comment:
Why here vs on the Stats struct? I don't have any preference. I'm just curious how we decide to put it on one or the other.